.circleci/config.yml

```diff
@@ -73,7 +73,7 @@ commands:
     steps:
       - run: wget -qO - https://download.opensuse.org/repositories/Emulators:/Wine:/Debian/xUbuntu_18.04/Release.key | apt-key add -
       - run: apt-get update || true && apt-get install -y ca-certificates libxtst-dev libpng++-dev && apt-get update && apt-get -y install << parameters.apt_opts >>
-
+      - run: wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 && chmod a+x /usr/local/bin/yq
```

Comment:
Can we pin the yq version instead of using latest? Osku is currently working on pinning versions everywhere across all CI pipelines and repositories, so we might as well be proactive on this, too :)

Same applies to the win and mac executors.